packages/core/src/mirrorworld.ts

```diff
@@ -2103,6 +2217,44 @@ export class MirrorWorld {
     return response.data.data;
   }
 
+  private async cancelEVMNFTListingTransaction(
+    payload: ICancelListingEVMPayloadV2
+  ): Promise<EVMNFTListingV2> {
+    assertEVMOnly('cancelEVMNFTListing', this.chainConfig);
+    const result = CancelEVMNFTListingSchemaV2.validate(payload);
+    if (result.error) {
+      throw result.error;
+    }
+    if (!result.value.from_wallet_address) {
+      throw new Error('from_wallet_address is required');
```

Comment:
`CancelEVMNFTListingSchemaV2.validate` should be able to validate this no?

Reply:
Problem is that it's optional there, but for this method it's not optional